.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
```

Comment:
could you also add black, trailing-whitespace, etc like here? https://github.com/uc-cdis/fence/blob/master/.pre-commit-config.yaml